tests/libstorage/datavolume.go

```diff
@@ -174,3 +174,7 @@ func HasDataVolumeCRD() bool {
 	}
 	return true
 }
+
+func HasCDI() bool {
+	return HasDataVolumeCRD()
```

Comment:
Do you see why we need `HasDataVolumeCRD` and why `HasCDI` is not enough?

Reply:
I tried to limit the PR to syntactic moves as much as possible to limit the scope.
I hope someone from the storage SIG can consider such a thing. @maya-r , @brybacki ?

Reply:
looks redundant, maybe there was no agreement on naming? I can take it

Reply:
The only think that I can think of about the original idea is this: One describes exactly what it does (checks that the CRD/CR is there) while the other expresses explicitly the interpretation of it.
In some way, it makes sense. But if the only thing that is publicly used is the interpretation ("has CDI"), then one can be a private function and the other public.